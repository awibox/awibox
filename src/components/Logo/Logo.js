import React from 'react';
// Styles
import styles from './Logo.scss';

const Logo = () => (
  <div className={styles.logo}>
    <div className={styles.bracket}>{'{'}</div>
    <div className={styles.text}>AWB</div>
    <div className={styles.bracket}>{'}'}</div>
  </div>
);


export default Logo;
