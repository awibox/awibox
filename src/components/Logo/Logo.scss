$logoSize: 32px;

.logo {
  display: flex;
  flex-flow: row;
  font-size: $logoSize;
  line-height: $logoSize;
  align-items: center;
}

.bracket {
  margin: -$logoSize/10 $logoSize/5 0;
  color: #0079c1;
  font-weight: 600;
}

.text {
  color: #fff;
}