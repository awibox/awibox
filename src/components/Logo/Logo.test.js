import React from 'react';
import { shallow } from 'enzyme';
import Logo from './Logo';

describe('Loader', () => {
  it('should render correctly', () => {
    const component = shallow(<Logo/>);
    expect(component).toMatchSnapshot();
  });
});
