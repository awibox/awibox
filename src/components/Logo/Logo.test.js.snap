// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should render correctly 1`] = `
<div
  className="logo"
>
  <div
    className="bracket"
  >
    {
  </div>
  <div
    className="text"
  >
    AWB
  </div>
  <div
    className="bracket"
  >
    }
  </div>
</div>
`;
